buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -51,9 +50,12 @@ if [ "${VANILLA_INSTALL}" != "yes" ]; then
     # ipykernel is used for testing ipython behaviours.
     if [ $PYTHON \< "3.12" ]; then
         $CONDA_INSTALL ${EXTRA_CHANNELS} cffi jinja2 ipython ipykernel pygments pexpect
-    else
+    elif [ $PYTHON \< "3.13" ]; then 
```

Comment:
It's more difficult to parse `elif` with a comparison condition that is a superset of the `if`, but it's fine to leave.